WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m

```diff
@@ -45,10 +45,26 @@ - (instancetype)initWithManagedObjectContext:(NSManagedObjectContext *)context m
         _aggregatedStatProperties = [[NSMutableDictionary alloc] init];
         _mixpanelProxy = mixpanelProxy;
         _context = context;
+
+        [self migrateSessionCountFromProxy: mixpanelProxy];
     }
     return self;
 }
 
+// Migrates the session count away from mixpanel super properties and into
+// Shared defaults.
+- (void)migrateSessionCountFromProxy:(MixpanelProxy *)mixpanelProxy
+{
+    NSInteger sessionCount = [WPAppAnalytics sessionCount];
+    if (sessionCount > 0) {
```

Comment:
It might make sense to have a "yup I migrated" key in user defaults rather than checking the current session count. It could be possible the notification for app did become active gets fired off before the Mixpanel tracker is brought up (although not likely with the current startup process).